srcpkgs/steam/files/README.voidlinux

```diff
@@ -28,8 +28,16 @@ installed for some games to function.
 If games are running slowly or not at all, or there are issues with network
 streaming, verify that your user belongs to the video group.
 
-If your audio is not working, try installing pulseaudio and
-alsa-plugins-pulseaudio.
+If your audio is not working, try installing pulseaudio,
+alsa-plugins-pulseaudio and their "<package>-32bit" pendants.
+
+If you try playing Windows games over SteamPlay/Proton and your game crashes
+with a diagnostic along the lines of "eventfd: Too many open files", PAM
+probably limits the open file descriptors per process a little to aggressively,
+inhibiting Proton's esync from spawning syncronization objects. 
+In this case consider editing "/etc/security/limits.conf" and setting a proper limit. 
+For more information see Valve's upstream documentation for Wine esync("README.esync") on GitHub:
+https://github.com/ValveSoftware/wine
```

Comment:
```suggestion
If you are using SteamPlay/Proton and the game crashes or hangs
with error messages that include "eventfd: Too many open files", the
number of open file descriptors per process is too low for complete Proton functionality. 
In this case, consult limits.conf(5) and set a higher nofile limit for your user. 
For more information, see upstream documentation  in the README.esync file from
https://github.com/ValveSoftware/wine
```

I think we can be more concise. And please fix line breaks so all lines have around the same length (in VIM you can do this by selecting the block and pressing `qg` while still in visual mode).